store/tikv/coprocessor.go

```diff
@@ -308,7 +308,7 @@ type copIterator struct {
 	errChan  chan error
 }
 
-const minLogCopTaskTime = 50 * time.Millisecond
```

Comment:
1 second is too long. We log slow query which take more than 300ms. How about set minLogCopTaskTime to 100ms?